content/altinn-studio/reference/subform/config-options/_index.en.md

```diff
@@ -8,3 +8,23 @@ weight: 120
 {{% notice warning  %}}
 This documentation is a work in progess. Subforms are currently in preview-release.
 {{% /notice %}}
+
+## Parameters
+
+| Parameter                                                                                                                 | Required | Description                                                                                                              |
+| ------------------------------------------------------------------------------------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------ |
+| id                                                                                                                        | Yes      | Unique ID, same as ID on other components. Must be unique in the layout file, and should be unique across pages.         |
+| type                                                                                                                      | Yes      | Must be 'Subform'                                                                                                        |
+| layoutSet                                                                                                                 | Yes      | The layout set to load for this subform, must be unique IDs within layout-sets.json.                                     |
+| [tableColumns](../../../../app/development/ux/fields/grouping/repeating/table/#widths-alignment-and-overflow-for-columns) | Yes      | Object containing column options for specified headers. If not specified, all columns will use default display settings. |
```

Comment:
We actually just invented our own tableColumns implementation here, it's not the same as repeating groups.